src/Filter/AbstractFilterer.php

```diff
@@ -77,7 +78,7 @@ public function filter(QueryCriteria $criteria, int $limit = null, int $offset =
         // END DEPRECATED BC LAYER
 
         foreach ($this->filterMutators as $mutator) {
-            $mutator($query, $actor, $criteria->query, $criteria->sort);
```

Comment:
So is actor not available anymore?

Reply:
It is: filter state is a wrapper around the query builder and the actor.